Documentation/ceph-csi-troubleshooting.md

```diff
@@ -441,3 +441,51 @@ $ rbd ls --id=csi-rbd-node -m=10.111.136.166:6789 --key=AQDpIQhg+v83EhAAgLboWIbl
 ```
 
 Where `-m` is one of the mon endpoints and the `--key` is the key used by the CSI driver for accessing the Ceph cluster.
+
+## Node Loss
+
+When a node is lost, you will see application pods on the node stuck in the `Terminating` state while another pod is rescheduled and is in the `ContainerCreating` state.
+
+To allow the application pod to start on another node:
+
+### Force delete your application pod. After some timeout (8-10 minutes), your pod will be rescheduled
+
+To force delete the pod stuck in the `Terminating` state:
+
+```console
+$ kubectl -n rook-ceph delete pod my-app-69cd495f9b-nl6hf --grace-period 0 --force
+```
+
+**And** then wait for timeout (8-10 minutes). If the pod still not in running state in thatcase , we need to wait more or go and blacklist the watcher(mentioned below).
+
+### Shorten the timeout
+
+To shorten the timeout, you can mark the node as "blacklisted" so Rook can safely failover the pod sooner.
```

Comment:
In case the node is just offline and there is no watcher active, we need to just blacklist the whole node, rather than blacklist just a session id, right? Seems like we could simplify this section to just blacklist the node ip.

Reply:
Response? I don't understand why we would want to only backlist only a session id instead of always blocklisting the whole node. The point is also to prevent a node from coming back online and create a new session, right?

Reply:
@Madhu-1 can you help here? Thanks

Reply:
We need to blacklist the IP as we want to block all sessions of that node.

Reply:
Ok, so if we want to blacklist all sessions, we only need the node ip, right? And no need to get the PV session IDs?

Reply:
Yes we just need the Node IP to blacklist it.